### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by lavdim** [frmichel/sparql-micro-service#11]
Hi,

could you please improve the documentation, in particular when creating a new SPARQL Endpoint, it should be obvious that the content defined in a Service Description file should be also added as a NAMEDGRAPH (the best with the same name as in the folder structure) in the Triple Store manually or by re-running the initial configuration file. Current, I am adding using the following SPARQL Insert query template:
```

PREFIXES

INSERT DATA {

GRAPH <http://localhost/sparql-ms/[ServiceName]/ServiceDescription> {

     Inner Content (the same what you would define in a ServiceDescription file)

}
```

**Comment by frmichel** [comment]
Yes you are correct, the documentation is not clear on that point. I will improve that.

Note that the [corese-server.sh file](https://github.com/frmichel/sparql-micro-service/blob/master/deployment/corese/corese-server.sh) builds a list of all the ServiceDescription* et ShapesGraph files to load them in our triple store (Corese-KGRAM) at startup. You may reuse that script and adapt it for Fuseki.

Franck.

**Comment by frmichel** [comment]
Fixed in commit [7750ec2](/frmichel/sparql-micro-service/commit/7750ec2ce38792fb933b8369c2bcc6285440a307).
Main changes in [/doc/04-install.md
